### ISSUE TITLE
Windows Pen

### ISSUE THREAD

**Issue by Ketsujiji** [krassowski/Anki-TouchScreen#2]
No support for Windows pen when using Surface products, have to keep button pressed and there is a small lag.

**Comment by fishnibble** [comment]
I saw the repo owner does not have own a windows pen. I own one and am willing to help out with what i can. I'm not familiar with the libraries that are out there that support the pen with python. @Ketsujiji Could you give me some ideas about what you've looked at so far?

**Comment by krassowski** [comment]
I think that [PointerEvent API](https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent) might be of some use, though I have not tried it.

**Comment by alaskakw** [comment]
@krassowski is a windows pen all you need? If it'd speed up the resolution of this problem, I'd be willing to buy one off ebay and have it shipped to you. Needless to say if you also need a surface, I wouldn't be able to help with that.

**Comment by krassowski** [comment]
No, I would need someone to rent the device or (much-preferred option) I would need someone to point me towards a way to set-up a virtual machine to test this issue. I did not find anything with a quick search, but I am not working with development for Windows and may miss keywords.

I know that Microsoft did such a thing back then when they introduced Edge - they released special VM images for developers to allow them to support their (back then) new browser.

**Comment by krassowski** [comment]
Or if someone is willing to try, I could guide them with some hints about how to implement this stuff.

**Comment by fishnibble** [comment]
I have a pen and a Windows Machine to support it

**Comment by krassowski** [comment]
Would you be able to test if this simple substitute of API to PointerEvents works?

Simply replace contents of the `__init__.py` file with the [this](https://raw.githubusercontent.com/krassowski/Anki-TouchScreen/feature/windows-pen/touchscreen/__init__.py). You can locate the file from Anki menu choosing `Tools` -> `Add-ons` -> select touchscreen addon -> `View Files` button; please remember to restart Anki afterwards.  Please let me know if it works for you.

**Comment by Ketsujiji** [comment]
I lost my account for github for a while, I tested the add-on and it registers the pen. will use it for a while to see how well it works

**Comment by AnthonyCouthures** [comment]
Hello here, 

I have the solution, this "bug" is due to the windows pen itself. In fact, it generates a lot more position data than a usual mouse and maybe other pen.

 I put the code below I use now and it seem to work well with my windows pen. This way I can choose the "step" of drawing for mor fluency. 

https://github.com/AnthonyCouthures/Touchscreen-Anki/blob/master/__init__.py
